packages/website/next.config.js

```diff
@@ -15,11 +16,9 @@ module.exports = withPlugins([withTM, withImages], {
     },
     webpack: (config, options) => {
         patchWebpackConfig(config, options);
-        if (process.env.NODE_ENV === 'development') {
-            config.watchOptions = {
-                poll: 10000,
-            };
-        }
```

Comment:
@aiheon that was causing a lot of delay, but I can revert. It was added when you worked on the search bar

Reply:
🤔  I don't event remember writing that 😅